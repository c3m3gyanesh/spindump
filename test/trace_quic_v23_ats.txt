A simple test of the ATS QUIC implementation, using regular 0-RTT connection setup, three separate connections. The client is Lsquic, the server is ATS. The following Lsquic commands were used:

   ./lsquic/http_client -H quic.ogre.com -s quic.ogre.com:4433 -p /en/latest/
   ./lsquic/http_client -H quic.ogre.com -s quic.ogre.com:4433 -p /en/latest/
   ./lsquic/http_client -H quic.ogre.com -s quic.ogre.com:4433 -p /en/latest/
