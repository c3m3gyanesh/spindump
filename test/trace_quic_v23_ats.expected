QUIC 172.16.114.251 <-> 67.192.246.159 0885055db8660c56-87b76604208c30804945e3ee9cc90d021949c9 (60946:4433) at 1571172448791915 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 0885055db8660c56-12b0118b3de3907d1285b2f0c865882ac0e4 (60946:4433) at 1571172449005785 measurement up right 62653 packets 2 0 bytes 2456 0 bandwidth 2456 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 85055db8660c56a4-12b0118b3de3907d1285b2f0c865882ac0e4 (60946:4433) at 1571172449366722 spinflip up 0-1 initiator packets 8 7 bytes 3354 8061 bandwidth 3354 8061 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 67.192.246.159 08dabb2e39a616e5-25b0e65e5432279c5de2ed75e9099f936913da (58420:4433) at 1571172452966520 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 08dabb2e39a616e5-125c7d0a473a6ff117381b1d2c7c8b06be3a (58420:4433) at 1571172453177211 measurement up right 59171 packets 2 0 bytes 2456 0 bandwidth 2456 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 dabb2e39a616e5c5-125c7d0a473a6ff117381b1d2c7c8b06be3a (58420:4433) at 1571172453538385 spinflip up 0-1 initiator packets 7 7 bytes 3267 8009 bandwidth 3267 8009 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 67.192.246.159 0885ebe28c34274f-bd18bf76e8861d56e687682c8517ae8acd6649 (61098:4433) at 1571172456391610 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 0885ebe28c34274f-120e649c91061595ba3fb6984f03234de78c (61098:4433) at 1571172456602137 measurement up right 58372 packets 2 0 bytes 2456 0 bandwidth 2456 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 67.192.246.159 85ebe28c34274f8d-120e649c91061595ba3fb6984f03234de78c (61098:4433) at 1571172456972621 spinflip up 0-1 initiator packets 7 16 bytes 3286 18827 bandwidth 3286 18827 note "V23,no R-spin"
